ragstack/colbert/colbert_embedding.py

```diff
@@ -161,50 +158,54 @@ def encode_queries(
             fixed_length = calculate_query_maxlen(
                 tokens,
                 max(query_maxlen, self.colbert_config.query_maxlen),
-                MAX_MODEL_TOKENS)
+                MAX_MODEL_TOKENS,
+            )
         # we only send one query at a time therefore tokens[0]
-        logging.info(f"{len(tokens[0])} tokens in first query with query_maxlen {fixed_length}")
+        logging.info(
+            f"{len(tokens[0])} tokens in first query with query_maxlen {fixed_length}"
+        )
 
         self.checkpoint.query_tokenizer.query_maxlen = fixed_length
 
         # All query embeddings in the ColBERT documentation
         # this name, EQ or Q, maps the exact name in most colBERT papers
-        Q = self.checkpoint.queryFromText(
+        queriesQ = self.checkpoint.queryFromText(
```

Comment:
```suggestion
        queries = self.checkpoint.queryFromText(
```

Reply:
Is there a way to align what's the variable in the paper and doc? I think the code should reflect what the paper describes. If Q cannot used, we should at least use queriesQ or something with a Q

Reply:
Then just `q` lowercase ?